contributors.md

```diff
@@ -1,10 +1,12 @@
 # Contributors
+
 1. [Flocea Dominic FAF-181](https://github.com/dominatorfaf)
 1. [Dragoș Străinu FAF-151](https://github.com/strdr4605)
-1. [Vdovicenco Alexandru FAF-151](https://github.com/ASV44) 
+1. [Vdovicenco Alexandru FAF-151](https://github.com/ASV44)
```

Comment:
Please don't include these changes in this PRs😊

Reply:
Fixed initial formatting. See [123f1f0](https://github.com/TUM-FAF/faf-contributions/pull/29/commits/123f1f03e930e7efaab4ee047ed4c66c84c262c6).